tests/zquantum/core/measurement_test.py

```diff
@@ -857,3 +858,81 @@ def test_get_measurements_representing_distribution_gives_exactly_right_counts(
         counts = measurements.get_counts()
         for bitstring, probability in bitstring_distribution.distribution_dict.items():
             assert probability * number_of_samples == counts[bitstring]
+
+    @pytest.mark.parametrize(
+        "bitstring_distribution",
+        [
+            BitstringDistribution(
+                {
+                    "0011": 0.0049,
+                    "1100": 0.0049,
+                    "1111": 0.008,
+                    "0000": 0.008,
+                    "0001": 0.008,
+                    "0010": 0.008,
+                    "0100": 0.008,
+                    "1000": 0.008,
+                    "1001": 0.008,
+                    "1010": 0.008,
+                    "1110": 0.008,
+                    "1101": 0.008,
+                    "0110": 0.9102,
+                }
+            )
+        ],
+    )
+    def test_get_measurements_representing_distribution_does_not_remove_bitstrings_not_sampled(
+        self, bitstring_distribution
```

Comment:
These lines caused stylechecks to [fail](https://github.com/zapatacomputing/z-quantum-core/runs/2823555534?check_suite_focus=true). The runner prints the failed command that you can invoke locally to replicate the issue:
```
flake8 --ignore=E203,E266,F401,W503 --max-line-length=88 src/python/zquantum/core steps tests/zquantum/core tests/acceptance_tests
```

Reply:
Fixed in c8135ca9b0a924a1aa8669e943bee4a7fc34fd19